public/locales/en/about.json

```diff
@@ -0,0 +1,55 @@
+{
+    "title": "About Us",
+    "placeholder": "Sample text change here. Vestibulum finibus ante vehicula, maximus erat vel, mollis ex. In erat elit, maximus at enim nec, tincidunt vulputate est. Quisque rhoncus egestas egestas. Vivamus ut iaculis ante, id volutpat lacus. Proin sit amet rutrum arcu. Cras elementum ligula nec volutpat varius. Maecenas viverra lectus quis velit finibus tempor. Donec et ligula enim. Sed eget scelerisque enim. Duis quis metus ac nisi accumsan eleifend id quis massa.",
+    "header": {
+        "brandAlt": "Crane logo with the text: Beneath the Surface",
+        "topics": "Topics",
```

Comment:
Do we need to repeat all this `header` stuff? My thought was that it would sit in header.json etc

Reply:
Another good catch, these don't need to be in this file. Right now they sit in `common.json` so I'll delete it here